qa/pom.xml

```diff
@@ -152,6 +152,11 @@
     <profiles>
         <profile>
             <id>vagrant</id>
+            <activation>
+                <property>
+                    <name>vagrant</name>
```

Comment:
Sure sure. Better idea. I'll do it when I get proper internet.
On Aug 10, 2015 5:38 AM, "Adrien Grand" <notifications@github.com> wrote:

> In qa/pom.xml
> <https://github.com/elastic/elasticsearch/pull/12769#discussion_r36626205>
> :
>
> > @@ -152,6 +152,11 @@
> >      <profiles>
> >          <profile>
> >              <id>vagrant</id>
> > +            <activation>
> > +                <property>
> > +                    <name>vagrant</name>
>
> +1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/12769/files#r36626205>.
>